### PR TITLE
Qualified import of reasoning modules fixing #2280

### PR DESCRIPTION
**Pull request by jamesmckinna**

### CODE REVIEW

File: src/Data/Integer/Divisibility/Signed.agda [hunk 1]
```diff
@@ -50,38 +49,38 @@ open _∣_ using (quotient) public
...
     ikq*∣k∣≢0 = *-nonZero ∣ ikq ∣ ∣ k ∣ {{ikq≢0}} {{≢-nonZero neq}}
 
   sign-eq : sign i ≡ sign (ikq * k)
   sign-eq = sym $ begin
-    sign (ikq * k)                  ≡⟨ sign-◃ (sign ikq S.* sign k) (∣ ikq ∣ ℕ.* ∣ k ∣) ⟩
-    sign ikq S.* sign k             ≡⟨ cong (S._* sign k) (sign-◃ (sign i S.* sign k) q) ⟩
-    (sign i S.* sign k) S.* sign k  ≡⟨ SProp.*-assoc (sign i) (sign k) (sign k) ⟩
-    sign i S.* (sign k S.* sign k)  ≡⟨ cong (sign i S.*_) (SProp.s*s≡+ (sign k)) ⟩
-    sign i S.* S.+                  ≡⟨ SProp.*-identityʳ (sign i) ⟩
+    sign (ikq * k)                  ≡⟨ sign-◃ (sign ikq Sign.* sign k) (∣ ikq ∣ ℕ.* ∣ k ∣) ⟩
```
Comment (JacquesCarette) [inline src/Data/Integer/Divisibility/Signed.agda:72]: picky: the alignment seems to need tweaked.

**Review by JacquesCarette — CHANGES_REQUESTED** [review]
A single thing needs tweaking. But also: #2270 doesn't document the names of various reasoning modules; you chose some that are very reasonable, but which do use unicode. That is one doubt that I have, i.e. did we agree on that? I don't remember! And neither the issue nor the change to the style guide bring clarity.

**Comment by jamesmckinna** [comment]
> A single thing needs tweaking. But also: #2270 doesn't document the names of various reasoning modules; you chose some that are very reasonable, but which do use unicode. That is one doubt that I have, i.e. did we agree on that? I don't remember! And neither the issue nor the change to the style guide bring clarity.

On the use of Unicode: we *had* agreed to the use of `ℕ` as the qualified module name for the (most-?) common cases of importing qualified from `Data.Nat.*`, and equality as `≡` from `Relation.Binary.PropositionalEquality.*`.

On the names chosen to standardise the various `Reasoning` modules, I needed to make *a* choice, moreover one which transferred across the imports as `≡-Reasoning`, `SetoidReasoning`, `PreorderReasoning`, `PartialOrderReasoning` (other, shorter names are also used). Following the various PRs to regularise the 'standard' infix notations for the various `Relation.Binary.Bundles`, I decided (unilaterally!) to use those symbols as the prefix to `-Reasoning`...

So yes: a design decision I 'sneaked' into the PR. I'll summarise the above on the main issue #2280 .

File: src/Data/Integer/Divisibility/Signed.agda (hunk 1 above)
Comment (jamesmckinna) [reply src/Data/Integer/Divisibility/Signed.agda]: Revised first commit fixes the alignment, but I wonder if this is a case where interleaving the terms and the reasoning line-by-line might be a better layout for the sake of avoiding long lines...?

Comment (jamesmckinna) [reply src/Data/Integer/Divisibility/Signed.agda]: The alignment in `Data.List.Relation.Binary.BagAndSetEquality` is also (historically) terrible, and I don't necessarily think this worth fixing here; against #1630 / #1930 , to my surprise this module also makes extensive use of `inspect`... so might want a separate consideration outside this 'boilerplate' PR...

Comment (jamesmckinna) [reply src/Data/Integer/Divisibility/Signed.agda]: Indeed: a global `cosmetic` issue on 'fixing alignment in reasoning' might be in order...

**Review by MatthewDaggitt — APPROVED** [review]

**Review by JacquesCarette — APPROVED** [review]

**Comment by JacquesCarette** [comment]
Right - 2 approvals for this design, so 3 people like it, seems good enough. Merging. It should be documented in the style guide.

**Pull request merged**